languages/CRB.groovy

```diff
@@ -0,0 +1,90 @@
+@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript
+import com.ibm.dbb.metadata.*
+import com.ibm.dbb.dependency.*
+import com.ibm.dbb.build.*
+import com.ibm.dbb.build.report.records.*
+import com.ibm.dbb.build.report.*
+import groovy.transform.*
+
+/*
```

Comment:
Have added this to the comments.
The script will be used to process CICS definition files from the repo with suffix .yml or .yaml.
The script will need the CICS definition file in YAML format, it will also need the Model file and 
the application constraints file.
The YAML formatted CICS definition files are created by the CICS TS resource builder tool.
Further information on the CICS TS resource builder tool and can be found at 
https://www.ibm.com/docs/en/cics-resource-builder/1.0?topic=overview
The link also gives a good explanation of the Model file and application constraints file and its usage.